File: 🟢 Easy/242 - Valid Anagram/_242Valid Anagram.js
/**
 * @param {string} s
 * @param {string} t
 * @return {boolean}
 */
var isAnagram = function (s, t) {
    if (s.length != t.length) {
        return false;
    }
    let sArr = s.split("");
    let tArr = t.split("");

    sArr.sort();
    tArr.sort();

    if (JSON.stringify(sArr) == JSON.stringify(tArr)) {
        return true;
    }

    return false;
};

File: 🟢 Easy/242 - Valid Anagram/_242Valid Anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        if (s.length() != t.length()) {
            return false;
        }

        char sArr[] = s.toCharArray();
        char tArr[] = t.toCharArray();
        Arrays.sort(sArr);
        Arrays.sort(tArr);

        if (Arrays.equals(sArr, tArr)) {
            return true;
        }

        return false;
    }
}